src/Echidna/Mapping/Nullability.cs

```diff
@@ -39,4 +39,16 @@ public static bool TryGetFor(PropertyInfo property, [NotNullWhen(returnValue: tr
         lock (Context) { nullabilityInfo = Context.Create(property); }
         return true;
     }
+
+    public static bool TryGetFor(FieldInfo field, [NotNullWhen(returnValue: true)] out NullabilityInfo? nullabilityInfo)
```

Comment:
@madelson Build was failing because of this; I just copypasted the method above and updated properties to fields. Does this look right?